docker-compose.yml

```diff
@@ -1,21 +1,18 @@
 version: "3.9"
 services:
-  minio:
-    image: lazybit/minio
+  localstack:
+    image: localstack/localstack:4.0
     ports:
-      - 9000:9000
+      - 4566:4566
     environment:
-      - MINIO_ACCESS_KEY=ACCESSKEY
-      - MINIO_SECRET_KEY=SECRETKEY
+      - SERVICES=s3,dynamodb,kms
+      - DEBUG=1
+      - LS_LOG=trace
```

Comment:
How much logging does trace end up being?  I could see it being useful but also maybe drowning test output.

Reply:
Whoops. I'm fine removing that.